ListView-and-Adapters-HW/app/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/activity_main"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:paddingBottom="@dimen/activity_vertical_margin"
+    android:paddingLeft="@dimen/activity_horizontal_margin"
+    android:paddingRight="@dimen/activity_horizontal_margin"
+    android:paddingTop="@dimen/activity_vertical_margin"
+    tools:context="com.elysium.listview_and_adapters_hw.MainActivity">
```

Comment:
I think I know why your buttons weren't showing up - the default orientation for a LinearLayout is horizontal, so unless you specify `android:orientation="vertical"` it will end up horizontal. Then your EditText has width "match_parent" so everything else ends up to the right of that, off the screen.